js/pages/SeatPage.ts

```diff
@@ -211,7 +211,9 @@ export class SeatPage extends PageBase {
         const soundManager = getSoundManager();
         soundManager.tableSoundsEnabled(false);
         if (!PageBlock.useDoubleView) {
+            orientationService.enableRotation();
```

Comment:
Why this is needed?

Reply:
Because now we suppres rotation when initialize app for all devices, in order to change slide width and update slider width during rotation, all rotations shoud be controled by us.

Reply:
The question appear here because it is starts polluting in the different areas. and on the different pages you don't add such code because they are always in default app orientation (portrait - phone, landscape - ipad). It is indication that our model of working with rotation has issues. 
What you do right now is you override `enableRotation` in specific places. So maybe it is better add parameter to 'setOrientation` so intent would be explicit.